.travis.yml

```diff
@@ -20,15 +20,17 @@ before_install:
   - sudo mv docker-compose /usr/local/bin
   - sudo docker --version
   - sudo docker-compose --version
-  - ./csu start
 
-install:
-  - pip3 install -r requirements/test.txt
+before_script:
+  - ./csu start
 
-# Runs test suite
 script:
-  - flake8
-  - pydocstyle --verbose --count --explain
+  - ./csu test
+  - ./csu test_backwards
+  - ./csu style
+
+after_script:
+  - docker-compose run django /docker_venv/bin/coveralls
```

Comment:
Still don't see this reflected on (see comment from review is probably outdated from this change):
https://coveralls.io/github/uccser/cs-unplugged